app/models/user.rb

```diff
@@ -116,6 +121,15 @@ def likes?(run)
     RunLike.find_by(user: self, run: run)
   end
 
+  # races will returns a standard ruby array, NOT an AR relation
+  def races
+    standard_races + bingo_races + randomizer_races
+  end
+
+  def in_race?
+    !races.all?(&:finished?)
```

Comment:
Can this be done in a query without loading all past races?

Reply:
See [this](https://github.com/glacials/splits-io/pull/550#discussion_r283105684)